### DISCUSSION TITLE
php 8.1 support

### DISCUSSION THREAD

**Discussion by ariaieboy** [livewire/livewire#4268] [7. Bugs (archived)] [answered]
hello 
php 8.1 will release publicly in less than a day.
it's very nice so we have php 8.1 support in livewire
i've added php8.1 test workflow in my fork and we have a failing unit test:
https://github.com/ariaieboy/livewire/runs/4304014745?check_suite_focus=true
the failed test is :https://github.com/livewire/livewire/blob/fe086248482ebcf7a3d749fa0a29c530de813e8b/tests/Unit/ComponentRootHasIdAndComponentDataTest.php#L33

do you guys have any idea that this test case cover which part of the livewire core ?>

**Comment by zepfietje** [comment, marked as answer]
Hey @ariaieboy, PHP 8.1 support has just been added in https://github.com/livewire/livewire/pull/4271.

It should be available in [Livewire version 2.8.0](https://github.com/livewire/livewire/releases/tag/v2.8.0).
